app/js/components/patientDashboard/View.jsx

```diff
@@ -56,7 +59,7 @@ const View = props => (
               </span>
               <span className="left-margin">
                 <span id="coreapps-telephoneNumber">
-                  {props.patient.attributes[0].value}
+                  {props.patient.attributes[0] ? props.patient.attributes[0].value : ''}
```

Comment:
How about using the short-circuit validation here as you used it on line 17?

Reply:
The desired behaviour is to show an empty string if the attribute is not present rather than taking it away altogether

Reply:
I see.
All good then.